python/hsfs/training_dataset.py

```diff
@@ -409,3 +419,6 @@ def get_statistics(self, commit_time=None):
             return self.statistics
         else:
             return self._statistics_engine.get(self, commit_time)
+
+    def query(self, storage="online"):
```

Comment:
from an api perspective, I think this could be a property, and then have a separate method `.get_query(storage)` see the statistics, how I did it there. not sure what's better.

Reply:
Ok, I see, let's keep it consistent then.

Reply:
fixed and added docstring